tests/gen/gen_auto.go

```diff
@@ -0,0 +1,251 @@
+// Copyright 2023 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package gen
+
+import (
+	"math/rand"
+
+	"github.com/sourcenetwork/defradb/client"
+	"github.com/sourcenetwork/defradb/client/request"
+)
+
+const (
+	// DefaultNumDocs is the default number of documents to generate for a collection.
+	DefaultNumDocs = 10
+	// DefaultNumChildrenPerDoc is the default number of children to generate for a document.
+	DefaultNumChildrenPerDoc = 2
+
+	// DefaultStrLen is the default length of a string to generate.
+	DefaultStrLen = 10
+	// DefaultIntMin is the default minimum value of an integer to generate.
+	DefaultIntMin = 0
+	// DefaultIntMax is the default maximum value of an integer to generate.
+	DefaultIntMax = 10000
+)
+
+type docRec struct {
```

Comment:
todo: I have no idea what a `docRec` is, please rename and/or add some documentation :)

Reply:
thanks for pointing this out. I turns out the structure is not needed, just the dockey.